src/EnergyPlus/DataRuntimeLanguage.hh

```diff
@@ -737,6 +737,11 @@ namespace DataRuntimeLanguage {
 struct RuntimeLanguageData : BaseGlobalStruct
 {
 
+    // In the API, we allow the user to manipulate user-defined EMS globals, but we skip the built-in ones to avoid
+    // problems.  This constexpr should be incremented if we ever add more built-ins so that we skip the right amount
+    // in the API calls.
+    static int constexpr NumBuiltInErlVariables = 27;
```

Comment:
OK, so create a single location constexpr where we declare the number of built-in EMS variables.